docs/docs/.vuepress/configs/envVars.js

```diff
@@ -39,5 +34,7 @@ module.exports = {
   TMP_SIZE_LIMIT: "2GB",
   DELAY_MIN_MAX_TIME:
     "You can pause your workflow for as little as one millisecond, or as long as one year",
-  PUBLIC_APPS: "1200",
+  PUBLIC_APPS: "1,200",
+  WARM_WORKERS_INTERVAL: "10 minutes",
```

Comment:
```suggestion
  WARM_WORKERS_INTERVAL: "10 minutes",
```

Reply:
Changed "10 minutes" to "10 minutes" to follow Google's style guide for numbers.